Sorting Algorithms/SortingAlgos/SortingAlgos.h

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#ifndef __SORTING_ALGOS_H
```

Comment:
I found an interesting link "https://stackoverflow.com/questions/34496418/is-define-banned-in-industry-standards"